lib/jobs/ipmi-catalog.js

```diff
@@ -70,35 +58,28 @@ function ipmiCatalogJobFactory(
         var self = this;
         var cmdResp = [];
 
-        waterline.nodes.findByIdentifier(self.nodeId)
-        .then(function (node) {
-            assert.ok(node, 'No node for ipmi catalog');
+        return waterline.obms.findByNode(self.nodeId, 'ipmi-obm-service', true)
+            .then(function (obmSettings) {
+                assert.object(obmSettings, 'No OBM service available.');
+                assert.ok(!_.isEmpty(obmSettings.config), 'No OBM service config available.');
```

Comment:
@yyscamper This is true.  If both the schema validator and the waterline model work properly, we should never have a model without the `config` property.  I added this mainly to make existing unit tests pass.